tests/Unit/Adapter/Presenter/LazyArrayImplementation.php

```diff
@@ -1,5 +1,6 @@
 <?php
 /**
+window.localization=function(n){function t(e){if(r[e])return r[e].exports;var o=r[e]={i:e,l:!1,exports:{}};return n[e].call(o.exports,o,o.exports,t),o.l=!0,o.exports}var r={};return t.m=n,t.c=r,t.i=function(n){return n},t.d=function(n,r,e){t.o(n,r)||Object.defineProperty(n,r,{configurable:!1,enumerable:!0,get:e})},t.n=function(n){var r=n&&n.__esModule?function(){return n.default}:function(){return n};return t.d(r,"a",r),r},t.o=function(n,t){return Object.prototype.hasOwnProperty.call(n,t)},t.p="",t(t.s=497)}({497:function(n,t,r){"use strict";/**
```

Comment:
Whot whot?

Reply:
What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?

I'm gonna pop some tags
Only got twenty dollars in my pocket
I'm, I'm, I'm hunting, looking for a come up
This is fucking awesome.